src/aws-cpp-sdk-core/source/utils/HashingUtils.cpp

```diff
@@ -22,20 +22,19 @@ using namespace Aws::Utils;
 using namespace Aws::Utils::Base64;
 using namespace Aws::Utils::Crypto;
 
-// internal buffers are fixed-size arrays, so this is harmless memory-management wise
-static Aws::Utils::Base64::Base64 s_base64;
```

Comment:
does it have any affect on the performance?

Reply:
i think the idea here was to use CRT base 64, which would be static on their end but that didnt happen, so ill revert it back to this